recommendations/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: page
+title: "Recommendations"
+---
+
+This section contains recommendations for JSON API implementations. These
+recommendations are intended to establish a level of consistency in areas that
+are beyond the scope of the base JSON API specification.
+
+## Recommendations for Naming <a href="#naming" id="naming" class="headerlink"></a>
+
+It is recommended that resource types, attribute names, and association
+names be "dasherized"; i.e. consist of only lower case alphanumeric
+characters and dashes (U+002D HYPHEN-MINUS, "-").
+
+It is also recommended that resource types be pluralized. Dasherized and
+pluralized resource types can be used as URL segments without translation,
+as discussed below.
+
+## Recommendations for URL Design <a href="#urls" id="urls" class="headerlink"></a>
+
+### Reference Document <a href="#urls-reference-document" id="urls-reference-document" class="headerlink"></a>
+
```

Comment:
This section nicely describes reference documents, but doesn't make an explicit recommendation. It probably should.